### ISSUE TITLE
LAN issues

### ISSUE THREAD

**Issue by KyrDerz** [Kron4ek/minecraft-vortex-launcher#59]
What would be the best way to play LAN with friends over this one?
I've managed to make a pretty decent modpack running:
1.18.2
Forge 40.1.76
OptiFine HD U H9 pre1

Would ZeroTear server with [Lan Server Properties](https://www.curseforge.com/minecraft/mc-mods/lan-server-properties) be enough?

I have been trying to make it run the whole day but nothing really worked.

**Comment by Kron4ek** [comment]
> Would ZeroTear server with Lan Server Properties be enough?

Yes, "Open to LAN" with online mode disabled should work. At least it works for me on a real LAN. And it's really important to disable online mode, otherwise other people will get the "Invalid Session" error when trying to connect to your server.

However, if "Open to LAN" does not work for you, you can run a dedicated Forge server with the same mods as on the client. Run the Forge installer and install the Forge server (select "Install server" instead of "Install client"), then run it once and change `online-mode=true` to `online-mode=false` in server.properties, then copy all your mods from the client to the server (copying the whole mods directory should be enough i guess, except maybe client-only mods).
